### DISCUSSION TITLE
Epit

### DISCUSSION THREAD

**Discussion by Abdouzster** [EU-ECDC/epitweetr#16] [3. Q&A] [answered]
I am trying to run epitweetr on Azure Windows VM and  the dependencies aborted several time with the following error.
![image](https://user-images.githubusercontent.com/23038328/113208890-5a91d100-9240-11eb-8585-978ddc54f8de.png)

**Comment by lauespinosa** [comment, marked as answer]
Please try the following steps:

1. Change winutils URL in the configuration page of epitweetr Shiny app to: https://github.com/steveloughran/winutils/blob/master/hadoop-3.0.0/bin/winutils.exe
2. Click on 'save settings' after changing the URL
3. Click on 'run dependencies'

**Reply by Abdouzster** [reply]
Thanks Laura! It is working
